core/src/test/java/com/backblaze/b2/json/B2JsonByteArrayOutputStreamTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+package com.backblaze.b2.json;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+
+import static junit.framework.Assert.assertNull;
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ * Unit test for B2JsonByteArrayOutputStream
+ */
+public class B2JsonByteArrayOutputStreamTest {
+
+    @Rule
+    public ExpectedException thrown  = ExpectedException.none();
+
+    @Test
+    public void testRequestedArraySizeOverTheMaxLimit() throws IOException {
```

Comment:
after we figure out what implementation will use for B2JsonByteArrayOutputStream, let's think about what we have to test here.